README.md

```diff
@@ -109,7 +114,7 @@ so if you want to change them, place yours on the same dir structure on your Rai
 1. `app/views/graphql_devise/mailer/reset_password_instructions.html.erb`
 
 The main reason for this difference is just to make it easier to have both Standard `Devise` and this gem running at the same time.
-Check [these files](app/views/graphql_devise/mailer) to see the available helper methods you can use in the views.
+Check [these files](app/views/graphql_devise/mailer) to see the available helper methods you can use in your views.
```

Comment:
I wasn't sure about "the views". "Your views" might not be appropriate here so maybe "the mailer views" or "your mailer views" ?

Reply:
I think your views is fine